hydra-access-controls/spec/spec_helper.rb

```diff
@@ -11,12 +11,22 @@
 
 require 'byebug' unless ENV['CI']
 
-if ENV['COVERAGE'] and RUBY_VERSION =~ /^1.9/
+def coverage_needed?
+  ENV['COVERAGE'] || ENV['TRAVIS']
+end
+
+if RUBY_VERSION =~ /^1.9/ && coverage_needed?
```

Comment:
we still test on this version of ruby?

Reply:
`hydra-head` 10.x currently claims support for 1.9: https://github.com/samvera/hydra-head/blob/master/hydra-head.gemspec#L17
I think changing supported ruby versions will come in 11.0 and I created a ticket for this: https://github.com/samvera/hydra-head/issues/441

Reply:
Okay!